pkg/tagger/collectors/static_main.go

```diff
@@ -0,0 +1,63 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+// +build docker kubelet
+
+package collectors
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/tagger/utils"
+)
+
+const (
+	staticCollectorName = "static"
+	staticExpireFreq    = 5 * time.Minute
+)
+
+// StaticCollector fetches "static" tags, e.g. those from an env var
+// It is not intended to run as a stand alone
+type StaticCollector struct {
+	infoOut      chan<- []*TagInfo
+	expire       *utils.Expire
+	lastExpire   time.Time
+	expireFreq   time.Duration
+	ddTagsEnvVar []string
+}
+
+func (c *StaticCollector) Detect(out chan<- []*TagInfo) (CollectionMode, error) {
+	var err error
+	c.infoOut = out
+	c.expire, err = utils.NewExpire(staticExpireFreq)
```

Comment:
Are these needed? Mind clarifying how expiration works here?

Reply:
you're right, they aren't needed - i pulled out the ComputeExpires part and forgot about this